R/data_color.R

```diff
@@ -147,6 +147,15 @@ data_color <- function(data,
   columns <- rlang::enquo(columns)
```

Comment:
(Sorry I can't get the comment closer) Can we do a match.args style for apply_to?

Reply:
Now added. Thanks.